src/content/documentation/publish/third-party-library-usage.md

```diff
@@ -92,6 +97,22 @@ Tip: If the library is on GitHub, you can usually find this version under the 
 
 {% capture content %}
 
+## Use of package managers
+
+Extensions developers can use package managers and package repositories like [npm](https://www.npmjs.com/) to retrieve third party libraries.
+
+With a default npm configuration, third party library dependencies are declared in the project's `package.json` file: this qualifies as a third party library link as [previously described](#how-to-determine-the-third-party-library-link).
+
+Reviewers must be be able to retrieve and review all packages used by your extension, so the use of private packages or non-public registries is not recommended. If you do, you will need to provide reviewers with access to the private packages. Access information should be provided along with build instructions as part of the [source code submission](https://extensionworkshop.com/documentation/publish/source-code-submission/) process.
```

Comment:
```suggestion
Reviewers must be be able to retrieve and review all packages used by your extension, so the use of private packages or non-public registries is not recommended. If you do, include those as "pre-installed" dependencies in your `node-modules` folder. Access information should be provided along with build instructions as part of the [source code submission](https://extensionworkshop.com/documentation/publish/source-code-submission/) process.
```

I am not sure if "pre-installed" is the right term here or descriptive enough. Thoughts on that wording?

Reply:
I don't think "pre-installed" will quite get the idea across. I took another shot at making this passage more explicit. WDYT?

```suggestion
Reviewers must be able to retrieve and review all packages used by your extension. As a result, the use of private packages or non-public registries is permissible but not recommended. If you use non-public dependencies, you will either need to include the relevant `node_modules` sub-directories in your [source code submission](/documentation/publish/source-code-submission/) or include access credentials and instructions in your build instructions.
```

Reply:
```suggestion
Reviewers must be able to retrieve and review all packages used by your extension. Therefore, the use of private packages or non-public registries is permissible but not recommended. If you use non-public dependencies, you must include the relevant `node_modules` sub-directories in your [source code submission](/documentation/publish/source-code-submission/) or access credentials and instructions in your build instructions.
```

Reply:
Either works for me, but let's remove the `or include access credentials and instructions in your build instructions.` part. We will need to consider and evaluate that first. Some developers have done it in the past by themselves and anecdotally, that lead to issues in reproducing the build.